lua/smart-splits/types.lua

```diff
@@ -40,6 +40,12 @@ local M = {
     ---@type SmartSplitsAtEdgeBehavior
     stop = 'stop',
   },
+  FloatWinBehavior = {
+    ---@type SmartSplitsFloatWinBehavior
```

Comment:
Thanks, added!

BTW I named the type `...FloatWin...` rather than `...FloatingWin...` to match the `float_win_behavior` key in the options table, but let me know if you have a preference for one over the other :wink: